### ISSUE TITLE
Stopping in STEP10 -> 10.mapsamples.pl. Program finished abnormally 

### ISSUE THREAD

**Issue by AntonioPuriel** [jtamames/SqueezeMeta#889]
I followed some previous recommendations to a similar problem, however it has not worked, it keeps stopping the process in the mapping.
[syslog.txt](https://github.com/user-attachments/files/16814989/syslog.txt)

**Comment by fpusan** [comment]
What error message did you get in stdout?

**Comment by fpusan** [comment]
Also if your dataset is not very big maybe you can share it with me and I can try running it in my computer and reproduce the error.

**Comment by fpusan** [comment]
And does the error still happen when you run the same command (same extra arguments as your real run, etc) but using the test dataset instead?

**Comment by fpusan** [comment]
I don't mean the `test_install.pl` script, but a SqueezeMeta run using the test fastq files that we redistribute with the databases.

Can you confirm that this has worked for you?

**Comment by fpusan** [comment]
The issue seems to be similar to #880.
 It could also be something related to writing permissions... did you perhaps run out of disk quota?

**Comment by AntonioPuriel** [comment]
I think you are right, I just got an email saying that I exceeded my disk quota

**Comment by fpusan** [comment]
Ok! Try running SqueezeMeta with this test dataset and the same options you were adding to your real run. It should run fast and not consume much space. If that works then we can be somewhat confident that SqueezeMeta runs well in your system and the problems are coming from elsewhere (e.g. disk quota)

**Comment by AntonioPuriel** [comment]
running "test_install.pl" ?

**Comment by fpusan** [comment]
> I don't mean the test_install.pl script, but a SqueezeMeta run using the test fastq files that we redistribute with the databases.
Can you confirm that this has worked for you?

No. When you (or someone else) download the SqueezeMeta databases there is also a test dataset that gets downloaded to the same location. The idea is to run SqueezeMeta on that to make sure everything is working ok.

**Comment by AntonioPuriel** [comment]
Ok, but how I can put that to run, thanks for helping me

**Comment by fpusan** [comment]
This test data should be present in the same place as the database.
When running `test_install.pl` it should say where the database is located.
So if the database is located at `/example/path/db` then the test data should be in `/example/path/test`.
There you will find a directory called `raw` with fastq files, and a couple of samples files formated for SqueezeMeta. From that you can run SqueezeMeta normally.

**Comment by fpusan** [comment]
Did you find this out?
Otherwise the file `/appli/conda-env/bioinfo/squeezemeta-1.6.3/SqueezeMeta/SqueezeMeta_conf.pl` has a line indicating where the databases are stored.

**Comment by fpusan** [comment]
What version of SqueezeMeta?
Can you attach the syslog file here?
Did you run trimming/quality filtering of your reads?

**Comment by fpusan** [comment]
Glad to hear! Closing issue

**Comment by zohrazahir** [comment]
Hi,
I faced the same problem:

[31mStopping in STEP10 -> 10.mapsamples.pl. Program finished abnormally
[0m[31m
  If you don't know what went wrong or want further advice, please look for similar issues in https://github.com/jtamames/SqueezeMeta/issues
  Feel free to open a new issue if you don't find the answer there. Please add a brief description of the problem and upload the /scratch/zohra125/SqueezeMeta_Results/meta_squeeze_output/syslog file (zip it first)
[0mDied at /home/zohra125/miniconda3/envs/SqueezeMeta/bin/SqueezeMeta.pl line 969.

Here are my syslog and my slurm.log:

[slurm-52518343.out.zip](https://github.com/user-attachments/files/18378618/slurm-52518343.out.zip)
[syslog.zip](https://github.com/user-attachments/files/18378619/syslog.zip)

I run from STEP 5 twice and both times it got killed at line 969


I ran the test. I rer-downloded and configured the DB. No file locations were changed, and the minimum RAM was always 250G. I am running this on a cluster (that gives me 20T storage). 

The only thing from my side: STEP 5 (Pfam) took about 4 days (it was very slow and was adding one bite every 2-6 sec) and so after 700MB I reached timeout and I didn't run it again, so in my restart, I skipped STEP5. 

I know you are busy, but any help would be great. 

Thank you,
Z.

**Comment by zohrazahir** [comment]
Thank you for the reply. 

Diamond ran 4 times: because three times I went out of time/our cluster went on maintenance/made a mistake :) and the fourth time it restarted correctly. 

- Memory: for the diamond part, I gave it 1T memory (but only used one node: each node 250G). Then I requested another node (our waitlist for a higher memory node was long) each 187G (but always 2-3 nodes) but I think it does not use multiple nodes. Could that be a problem? I can request a node with higher memory.
